packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -236,13 +239,23 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
                                 data-ng-bind-html="'pci_project_instances_instance_add_region_activate' | translate: {
                                     regionsLink: $ctrl.regionsLink
                                 }"
+                                data-ng-if="!$ctrl.isDiscoveryProject"
                             ></p>
                             <p
                                 class="warning-text"
                                 data-ng-switch-default
                                 data-ng-bind-html="'pci_project_instances_instance_add_region_quota' | translate: {
                                     quotaLink: $ctrl.quotaLink
                                 }"
+                                data-ng-if="!$ctrl.isDiscoveryProject"
+                            ></p>
+                        </div>
+                        <div data-ng-if="$ctrl.isDiscoveryProject ">
+                            <p
+                                class="warning-text"
+                                data-ng-bind-html="'pci_project_instances_instance_activate_project_banner_message' | translate: {
+                                    goToDiscoveryProjectActivationPage: $ctrl.goToDiscoveryProjectActivationPage
+                                }"
```

Comment:
This can lead to CSTI attacks
You must separate the `a` tag from the translation